docs/reverse_proxy.md

```diff
@@ -79,6 +79,10 @@ server {
         # Nginx by default only allows file uploads up to 1M in size
         # Increase client_max_body_size to match max_upload_size defined in homeserver.yaml
         client_max_body_size 50M;
+	
+	# Synapse responses may be chunked, which is an HTTP/1.1 feature.
+	proxy_http_version 1.1;
+        proxy_buffering off;
```

Comment:
This is also due to the face we're doing long polling; see this post https://www.nginx.com/blog/avoiding-top-10-nginx-configuration-mistakes/#proxy_buffering-off

Reply:
What difference would we expect to see with and without `proxy_buffering`?
The only long polling I can think of is /sync, which is supposed to time out at 30 seconds.

Reply:
That's a fair note, I'll remove that so as not to muddy the main point of the PR.